packages/dtslint/test/index.test.ts

```diff
@@ -70,93 +67,16 @@ describe("dtslint", () => {
     });
   });
   describe("rules", () => {
-    const tests = readdirSync(testDir).filter((x) => x !== "index.test.ts");
+    const tests = readdirSync(testDir); // .filter((x) => x !== "index.test.ts");
     for (const testName of tests) {
       const testDirectory = join(testDir, testName);
       if (existsSync(join(testDirectory, "tslint.json"))) {
         testSingle(testDirectory);
-      } else {
+      } else if (statSync(testDirectory).isDirectory()) {
         for (const subTestName of readdirSync(testDirectory)) {
           testSingle(join(testDirectory, subTestName));
```

Comment:
I don't understand how `testSingle` is testing eslint files, since it delegates most of its work to tslint's consoleTestResultHandler and runTest. Are the eslint tests still getting run?

Reply:
They are; Jest picks up any `test/**/*.test.ts` file in all packages based on the Jest config: https://github.com/microsoft/DefinitelyTyped-tools/blob/8b1430298dfffc2e235af5515f50094dfc512544/jest.config.js#L5

The CI output has a mention (https://github.com/microsoft/DefinitelyTyped-tools/runs/7754177707?check_suite_focus=true#step:7:17):

```plaintext
PASS packages/dtslint/test/dt-header.test.ts
```

Reply:
cool, so this file can be deleted when the last of the tslint tests is removed!